grpc/src/main/java/com/linecorp/armeria/server/grpc/UnframedGrpcErrorHandler.java

```diff
@@ -114,7 +106,15 @@ static UnframedGrpcErrorHandler ofJson(UnframedGrpcStatusMappingFunction statusM
                                                                    .addInt(GrpcHeaderNames.GRPC_STATUS,
                                                                            grpcCode.value())
                                                                    .build();
-            return HttpResponse.ofJson(responseHeaders, message);
+            final ImmutableMap.Builder<String, String> messageBuilder = ImmutableMap.builder();
+            messageBuilder.put("grpc-code", grpcCode.name());
+            if (grpcMessage != null) {
+                messageBuilder.put("message", grpcMessage);
+            }
+            if (cause != null && ctx.config().verboseResponses() && !status.isOk()) {
```

Comment:
nit; I believe this whole code block is triggered if `status != OK`.

https://github.com/line/armeria/blob/088202fea3ceab2408da2eaa1f763242135e76ea/grpc/src/main/java/com/linecorp/armeria/server/grpc/AbstractUnframedGrpcService.java#L180

```suggestion
            if (cause != null && ctx.config().verboseResponses()) {
```